dashboard/test/models/teacher_score_test.rb

```diff
@@ -12,14 +12,16 @@ class TeacherScoreTest < ActiveSupport::TestCase
     @section.add_student(@student_2)
     @section.add_student(@student_3)
     @script = create :script
+    @lesson_group = create :lesson_group, script: @script
+    @stage = create :lesson, script: @script, lesson_group: @lesson_group
     @script_level = create(
       :script_level,
       script: @script,
+      lesson: @stage,
```

Comment:
one idea for a shortcut here would be to make the `lesson` and `script_level` factories auto-create the lesson group, kind of like `script_level` is auto-creating the `lesson` in the left hand side of this diff. please only do if this makes life easier for you.